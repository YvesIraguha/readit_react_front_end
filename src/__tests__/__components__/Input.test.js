import React from "react";
import { shallow } from "enzyme";
import Input from "../../components/common/Input";

describe("input component", () => {
  test("should render the input field correctly", () => {
    const props = {
      className: "hell world"
    };
    const wrapper = shallow(<Input {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
