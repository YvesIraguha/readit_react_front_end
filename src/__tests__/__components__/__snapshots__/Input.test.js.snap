// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input component should render the input field correctly 1`] = `
<input
  className="hell world"
/>
`;
